OrbitGl/EventTrack.h

```diff
@@ -11,7 +11,7 @@
 class GlCanvas;
 class TimeGraph;
 
-class EventTrack : public Track {
+class EventTrack : public virtual Track {
```

Comment:
I'm currently trying to figure out why virtual inheritance is needed. Can you briefly explain that?

Reply:
I though I needed it to be able to inherit from std::enable_shared_from_this<Track> in "Track", but apparently I was mislead by another compiler error, because this should only be needed with multiple inheritance... looks like I can remove this again, thanks.